api/python/provisioner/utils.py

```diff
@@ -218,3 +222,26 @@ def repo_tgz(
     run_subprocess_cmd(cmd)
 
     return dest
+
+
+# Validates the hostname string in config.ini
+# against hostname in CLI args
+def node_hostname_validator(
+    nodes: List,
+    config_path: str
+):
+    logger.debug(f"Validating list of nodes: {'srvnode-1 : '.join(nodes)}")
```

Comment:
nodes  is list of Node class object from setup_provisioner so we need to convert that Node object list to list of hostnames.
`nodes = [node.host for node in nodes]`